index.qmd

```diff
@@ -188,11 +197,12 @@ Notas.
 -   `MinFD`: Diâmetro mínimo de corte, em centímetros
 -   `UpMinFD`: Diâmetro mínimo de corte aprimorado (caso de povoamento excessivamente rico), em centímetros
 -   `MaxFD`: Diâmetro máximo de corte, em centímetros
--   `Aggregative`: Natureza agregadora da espécie para converter árvores isoladas
+-   `Aggregative`: Natureza agregadora da espécie para converter árvores isoladas <!-- not sure converter is the right word... evitar ?
+    Also, I'm not sure we want to explain this as it may take quite some time and it is not central... what do you think?-->
 :::
 
 ::: notes
-Notas.
+Explain here what are the different level of CommercialLevel, as there may not be different level in Brasil.
```

Comment:
Only as a note or also in text?

Reply:
maybe we say it (so as a note) but we do'nt write it on the slide?